packages/account/src/test-utils/seedTestWallet.ts

```diff
@@ -5,20 +5,29 @@ import { coinQuantityfy, ScriptTransactionRequest } from '../providers';
 import type { CoinQuantityLike } from '../providers';
 import { WalletUnlocked } from '../wallet';
 
-export const seedTestWallet = async (wallet: Account, quantities: CoinQuantityLike[]) => {
+export const seedTestWallet = async (
+  wallet: Account | Account[],
+  quantities: CoinQuantityLike[],
+  utxosAmount = 1
+) => {
+  const toFundAccounts = Array.isArray(wallet) ? wallet : [wallet];
+
   const genesisWallet = new WalletUnlocked(
     process.env.GENESIS_SECRET || randomBytes(32),
-    wallet.provider
+    // Connect to the same Provider as wallet
+    toFundAccounts[0].provider
```

Comment:
This is a bit less intuitive than `wallet.provider`
```suggestion
   // There may be multiple wallets, so want to use the same provider for them all
   const walletProvider = toFundAccounts[0].provider
 ```

Reply:
@maschad do you think it is better now? https://github.com/FuelLabs/fuels-ts/pull/2037/commits/a657ee5ea356626a0312504239beeb419b5a8c01